Bigtable/tests/System/ChainFilterTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Filter;
+use Google\Cloud\Bigtable\Tests\System\FilterTest;
+
+/**
+ * @group bigtable
+ * @group bigtabledata
+ */
+class ChainFilterTest extends FilterTest
```

Comment:
What is the reason for separate files for each type of test, as opposed to grouping them in `FilterTest`? It looks like may even be able to DRY things up a bit using a dataprovider fed data from a file if you went that route. WDYT?